scheduler/domain/setup/api.go

```diff
@@ -2,11 +2,11 @@ package setup
 
 import (
```

Comment:
Echoing offline comment I made, but let's move setup to the scheduler top level so domain has no subpackages.